config/mongoid.yml

```diff
@@ -1,19 +1,18 @@
 development:
   sessions:
     default:
-      database: <%= ENV['MONGO_DB_NAME'] || 'content_store_development' %>
-      hosts:
-        - localhost:27017
+      # Contains draft_content_store_development or content_store_development
+      # depending on whether we're running content store in draft mode or not.
+      uri: <%= ENV['MONGODB_URI'] %>
```

Comment:
fair point @alext. i've updated to have a fallback db name. i've changed the comment to be unambiguous.